Makefile

```diff
@@ -85,16 +85,16 @@ TESTS=.
 TE_PACKAGES=$(shell $(GO) list ./...)
 
 # Plugins Packages
-PLUGIN_PACKAGES?=mattermost-plugin-zoom-v1.3.0
-PLUGIN_PACKAGES += mattermost-plugin-autolink-v1.1.2
+PLUGIN_PACKAGES?=mattermost-plugin-zoom-v1.3.1
+PLUGIN_PACKAGES += mattermost-plugin-autolink-v1.2.0
```

Comment:
Given that autolink `v1.2.0` contains a breaking change that we want to fix, I'm 3/5 that we should not force an upgrade.

Reply:
Thanks @hanzei.  I reverted to the previous version.